src/test/java/com/joss/conductor/mobile/LocomotiveTest.java

```diff
@@ -303,6 +303,68 @@ public void call() throws Throwable {
         }).isInstanceOf(IllegalArgumentException.class);
     }
 
+    @Test
+    public void test_perform_corner_swipe() {
+        WebDriver.Window window = mock(WebDriver.Window.class);
+        when(window.getSize()).thenReturn(new Dimension(414, 736)); // iPhone 6 Plus
+
+        WebDriver.Options options = mock(WebDriver.Options.class);
+        when(options.window()).thenReturn(window);
+
+        when(mockDriver.manage()).thenReturn(options);
+
+        final Locomotive locomotive = new Locomotive(androidConfig, mockDriver);
+
+        // Swipe Up Bottom Right Corner
+        locomotive.swipeCornerLong(ScreenCorner.BOTTOM_RIGHT, SwipeElementDirection.UP, 100);
+        locomotive.swipeCornerSuperLong(ScreenCorner.BOTTOM_RIGHT, SwipeElementDirection.UP, 100);
+        verify(mockDriver, times(1))
+                .swipe(404, 726, 404, 358, 100);
+        verify(mockDriver, times(1))
+                .swipe(404, 726, 404, /* ~ x - 1 to avoid going off screen ~ */1, 100 );
+
+        // Swipe Up Bottom Left Corner
```

Comment:
Personally I'd like these tests broken out into seperaet tests, but I understand you're following the previous example.